File: euler/2.cpp
#include <bits/stdc++.h>
using namespace std;
//https://projecteuler.net/problem=2
int main(){
        long long sum=2; //3 if not even
        long long b=1;
        long long a=2;
        long long temp;
        for(long long i=0;i<4000000-2;i++){
                temp=a;
                a=a+b;
                b=temp;
                if(a%2==0)
                        sum+=a;
                //cout<<a<<'\n';
        }
        cout<<sum;
}

File: euler/2.py
#https://projecteuler.net/problem=1
sum=2
b=1;
a=2;
temp=0
for i in range(4000000-2):
    temp=a;
    a=a+b;
    b=temp;
    if(a%2==0):
        sum+=a;
print(sum)
